docs/design.md

```diff
@@ -232,8 +232,8 @@ maximum-image-size = image-slot-size - image-trailer-size
 ```
 
 Where:  
-  `image-slot-size` is the size of the image slot.
-  `image-trailer-sectors-size` is the size of the image trailer.
+  `image-slot-size` is the size of the image slot.  
```

Comment:
Maybe remove the trailing space here?

Reply:
I added them intentionally. The next sentence begins from the new line in GH rich text view thanks to that.